outputs/mixins.py

```diff
@@ -635,7 +636,11 @@ def write_data(self, worksheet):
         self.write_header(worksheet, fields, iterative_sets_fields)
 
         # write content
-        self.write_content(worksheet, fields, iterative_sets_fields, objects)
+        paginator = Paginator(objects, 1000)
+
+        for page_number in paginator.page_range:
+            page = paginator.page(page_number)
+            self.write_content(worksheet, fields, iterative_sets_fields, page.object_list)
```

Comment:
I'm afraid it will override the existing rows. See the implementation of the write_content() method:

```
    def write_content(self, worksheet, fields, iterative_sets_fields, objects):
        # Write actual data. Start from the first cell. Rows and columns are zero indexed.
        row = 1
        max_col = 0
```

Reply:
thanks, I didn't notice